package-lock.json

```
@@ -17365,7 +17365,7 @@
         "@graphql-codegen/typescript": "^1.23.0",
         "@graphql-codegen/typescript-graphql-request": "^4.5.5",
         "@graphql-codegen/typescript-operations": "^1.18.4",
-        "@nft/api-graphql": "^1.0.0-beta.7",
+        "@nft/api-graphql": "1.0.0-beta.9",
```

This change should be indicated in the "changed" section of the changelog.
Also you didn't run `npm install` after, it will change this line to:
```suggestion
        "@nft/api-graphql": "^1.0.0-beta.9",
```

---

done